core/dev/mysql/mysql-deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: mysql
+  name: mysql
```

Comment:
@bradsawadye Please make this name a bit more verbose. Theoretically, we could have multiple mySql databases relating to independent packages. Being able to easily distinguish between them will become essential as the project ages.

@BMartinos what do you think the name should be? In my branch, I was perhaps too verbose.
`hapi-fhir-mysql-deployment`
I specified `hapi` because we also specified the `openhim` in its deployments such as `openhim-mongo...`